app/controllers/api/items_controller.rb

```diff
@@ -0,0 +1,74 @@
+module API
+  class ItemsController < ApplicationController
+    before_filter :validate_item, only: [:create, :update]
+    before_filter :authorize_token
+    before_filter :my_load_and_authorize_resource, only: [:update, :destroy, :show]
+
+    attr_reader :item
+
+    def create
+      head :created, location: sufia.api_item_path(actor.create_file_from_item)
+    end
+
+    def update
```

Comment:
@inukshuk I thought we were tossing a 404 in responses to `update` and `delete` if the specified Sufia ID it not found.  See here: https://github.com/projecthydra/sufia/pull/1022/files#diff-b2d6cad020de1805df8ddd050e8e159fR40  If that's not catching the exception and instead a 500 is being raised, that'd be good to know. If so, can you tell me how I can reproduce this?